transitions/registration.js

```diff
@@ -154,17 +154,17 @@ module.exports = {
                     return;
                 }
                 if (event.name === 'on_create') {
-                    var args = [db, doc];
-                    if (event.params) {
-                        // params setting get sent as array
-                        args.push(event.params.split(','));
-                    }
                     var obj = _.defaults({}, doc, doc.fields);
                     if (self.isBoolExprFalse(obj, event.bool_expr)) {
                         return;
                     }
+                    var options = { db: db, audit: audit, doc: doc };
+                    if (event.params) {
+                        // params setting get sent as array
+                        options.params = event.params.split(',');
+                    }
                     series.push(function(cb) {
```

Comment:
Agreed. Didn't want to update any more than necessary. `cb` is used everywhere here  :(